Kitodo/src/main/java/de/sub/goobi/helper/GoobiScript.java

```diff
@@ -179,11 +179,11 @@ public void execute(List<Process> processes, String script) throws DataException
     private void updateContentFiles(List<Process> processes) {
         for (Process proz : processes) {
             try {
-                Fileformat myRdf = serviceManager.getProcessService().readMetadataFile(proz);
+                FileformatInterface myRdf = serviceManager.getProcessService().readMetadataFile(proz);
                 myRdf.getDigitalDocument().addAllContentFiles();
                 serviceManager.getFileService().writeMetadataFile(myRdf, proz);
                 Helper.setMeldung("kitodoScriptfield", "ContentFiles updated: ", proz.getTitle());
-            } catch (ugh.exceptions.DocStructHasNoTypeException e) {
+            } catch (org.kitodo.api.ugh.exceptions.DocStructHasNoTypeException e) {
```

Comment:
Add this exception in imports.

Reply:
This exception is already imported but wrong used. Cleaning up would be nice.